.github/workflows/main.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
+    - name: Install portaudio system dependency
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y portaudio19-dev
```

Comment:
The rosdep installation and python requirement installation are currently separated in CI, therefore I think the one in rosdep won't be installed for the second task in CI, right?

Reply:
The rosdep install should be first and I think since recently it is done in both CI tasks.

Reply:
This is what we have in the second CI task, I can not find where is the installation of rosdep. So at least not in this branch.

    build_python_requirements:
      runs-on: ubuntu-20.04
      steps:
      - uses: actions/checkout@v3
      - name: Install portaudio system dependency
        run: |
          sudo apt-get update
          sudo apt-get install -y portaudio19-dev
      - name: Set up Python 3.8.10
        uses: actions/setup-python@v4
        with:
          python-version: 3.8.10
      - name: Install dependencies
        run: |
          python -m pip install --upgrade pip
          pip install wheel
          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi

Reply:
Ahh nvm I somehow missunderstood the description of #253 as it only manually added one dependency. Personally I would like to include the python deps into the main CI task, but this might be a separate topic.